stroyprombeton/tests/tests_selenium_admin.py

```diff
@@ -479,8 +484,10 @@ def test_save_and_drop_custom_filters(self):
 
         for index, item in enumerate(checkboxes):
             self.browser.find_elements_by_class_name('filter-fields-item')[index].click()
+            wait(0.5)
 
         self.save_filters()
+        wait(2)
```

Comment:
It isn't a stable approach. As i remember `save_filters` reloads the page, so it is better to wait the page to load